src/tests/ftest/nvme/nvme_health.py

```diff
@@ -30,21 +30,18 @@ def test_monitor_for_large_pools(self):
         # pylint: disable=attribute-defined-outside-init
         # pylint: disable=too-many-branches
         no_of_pools = self.params.get("number_of_pools", '/run/pool/*')
-        #Stop the servers to run SPDK too to get the server capacity
-        self.stop_servers()
-        storage = self.get_nvme_max_capacity()
-        self.start_servers()
+        storage = self.get_max_storage_sizes()
 
-        #Create the pool from 80% of available of storage space
-        single_pool_nvme_size = int((storage * 0.80)/no_of_pools)
+        #Create the pool from 90% of available of storage space
+        single_pool_nvme_size = int((storage[1] * 0.90)/no_of_pools)
```

Comment:
Just a thought... Can we move the percentage to YAML file parameter? In this way, you get modify in the YAML for future without need to update the script.
eg:
fill_max_pool_capacity : 90

Reply:
It's not major as both the cases it has to change, but we can update later as I do not want to restart the full Ci pipeline which will take another day to finish.

Reply:
Sure. No problem with me...

Reply:
With all tests passed. I think we should table the changes later... I didn't see that CI has completed.

Reply:
Updated the test and added fill percentage in yaml.